File: backend/src/main/java/com/hyunprise/backend/domain/member/vo/Member.java
package com.hyunprise.backend.domain.member.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Timestamp;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Member {
    private String memberUUID;
    private Long authenticationId;
    private String memberEmail;
    private String memberName;
    private Integer hPoint;
    private Integer gender;
    private Timestamp birthdate;
    private String address;
    private String contact;
    private Integer accountType;
    private Long brandId;
}

File: android/app/src/main/java/com/hyunprise/android/api/member/vo/Member.kt
package com.hyunprise.android.api.member.vo

import java.sql.Timestamp

data class Member (
    val memberUUID: String? = null,
    val memberEmail: String? = null,
    val memberName: String? = null,
    val membershipPoint: Int? = null,
    val gender: Int? = null,
    val birthdate: Timestamp? = null,
    val address: String? = null,
    val contact: String? = null,
    val accountType: Int? = null,
    val brandId: Long? = null
)